docs/concepts/what-is-filecoin.md

```diff
@@ -0,0 +1,27 @@
+---
+title: What is Filecoin?
+---
+
+# What is Filecoin?
```

Comment:
Since Filecoin has a ton of definitions (the storage network, the token, etc.), should we change this header to "What is the Filecoin storage network?" or something similar?

Reply:
Good point though I'd rather fix this a different way: keep the more general title, and push the header/page to be more general. I think this could be done with just a couple key points, which might fit nicely after 2nd paragraph & before diagram:

- Miners earn units of the Filecoin cryptocurrency (FIL).
- Transactions to send and receive FIL are also recorded in the Filecoin blockchain

For reference you may want to check out the first 2 paragraphs of the [Specification Intro](https://filecoin-project.github.io/specs/#intro), though it's more technical than we want for a concept doc.

Reply:
Here is the paragraph I added to introduce FIL and the blockchain:

> Filecoin includes a blockchain and native cryptocurrency (FIL). Storage miners earn units of FIL for storing files. Filecoin’s blockchain records transactions to send and receive FIL, along with proofs from storage miners that they are storing their files correctly.